src/Parser/Core/Modules/CastEfficiency.js

```diff
@@ -81,6 +81,56 @@ class CastEfficiency extends Analyzer {
     };
   }
 
+  /*
+   * Time spent casting the ability
```

Comment:
This comment doesn't add any information and can be omitted or changed to explain why or summarize what it does